frontend/src/scenes/session-recordings/player/player-meta/PlayerMetaBottomSettings.tsx

```diff
@@ -84,6 +91,24 @@ function InspectDOM(): JSX.Element {
     )
 }
 
+function Screenshot(): JSX.Element {
+    const { sessionPlayerMetaData } = useValues(sessionRecordingPlayerLogic)
+    const { takeScreenshot } = useActions(sessionRecordingPlayerLogic)
+
+    return (
+        <SettingsButton
```

Comment:
![2025-05-21 10 02 40](https://github.com/user-attachments/assets/47883115-e367-4bcf-a4b0-8b94a613a99e)

i don't know if it's related to this change or the bar in general...